invenio_matcher/core.py

```diff
@@ -24,78 +24,57 @@
 
 """Matcher core."""
 
-from invenio.ext.es import es
+import copy
 
-from .errors import InvalidQuery
-from .utils import build_exact_query, build_fuzzy_query
+from invenio_matcher.engine import exact
```

Comment:
Combine all these imports into one statement, and make it relative ideally. E.g.

```python
from .engine import (
    one,
    two,
    ...
)
```